### ISSUE TITLE
elm-format unable to determine elm version

### ISSUE THREAD

**Issue by winduptoy** [elm-community/SublimeElmLanguageSupport#53]
Nothing was working with this package in ST3, so I enabled debug mode and opened the console.

The latest version of `elm-format` (0.8.1-rc2) is throwing this:

```
Elm Language Support: (elm-format) b"I couldn't figure out what version of Elm you are using!\n\nYou should either run elm-format from the directory containing\nyour elm.json (for Elm 0.19) or elm-package.json (for Elm 0.18),\nor tell me which version of Elm with --elm-version=0.19 or --elm-version=0.18" 
```

I have no way to specify which version of elm my project is using (0.19) and it's not picking up on it automatically (maybe it's not being executed in the correct path?). I've downgraded to `elm-format-0.19-0.8.0` which is working.

**Comment by sentience** [comment]
Thanks for the report! I'll try to find some time to take a look at this during the coming week.

**Comment by sentience** [comment]
This is fixed in [1.0.0 beta 2](
https://github.com/elm-community/SublimeElmLanguageSupport/releases/tag/1.0.0-beta.2).
